chemicaltoolbox/autodock_vina/prepare_box/prepare_box.xml

```diff
@@ -0,0 +1,37 @@
+<tool id="prepare_box" name="Calculate the box parameters for an AutoDock Vina job from an input mol file (confounding box)" version="0.1.0">
+    <requirements>
+        <requirement type="package">rdkit</requirement>
```

Comment:
please include versions here as well for reproducibility

Reply:
resolved by c0d9ecf83a9a57b4011d4e47769ed950fd5ee798